extra/yoda.py

```diff
@@ -48,6 +48,7 @@
   --bus <bus>              Filter devices by bus
   --address <address>      Filter devices by address in bus
   --usb-port <port>        Filter devices by USB port in bus
+  --unsafe <flags>         Specify comma separated unsafe flags for use with unstable drivers (some coreliquid coolers)
```

Comment:
While it's being ported to yoda because of the untable coreliquid coolers, the option is (or may be) useful with other devices too. I think the description from the CLI is good/better:

```suggestion
  --unsafe <features>         Comma-separated bleeding-edge features to enable
```